hq/test/schedule/SendTestNotifications.scala

```diff
@@ -1,78 +0,0 @@
-package schedule
-
-import com.amazonaws.ClientConfiguration
-import com.amazonaws.auth.profile.ProfileCredentialsProvider
-import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
-import com.amazonaws.services.sns.{AmazonSNSAsync, AmazonSNSAsyncClientBuilder}
-import com.gu.anghammarad.models.{Notification, AwsAccount => Account}
-import config.Config
-import model._
-import org.joda.time.DateTime
-import org.scalatest.{DoNotDiscover, FreeSpec, Matchers}
-import org.scalatestplus.play.components.OneAppPerSuiteWithComponents
-import play.api.routing.Router
-import play.api.{BuiltInComponents, BuiltInComponentsFromContext, NoHttpFiltersComponents}
-import schedule.IamMessages.VulnerableCredentials.disabledUsersMessage
-import schedule.IamMessages.disabledUsersSubject
-import schedule.Notifier.notification
-import utils.attempt.{Attempt, AttemptValues}
-
-import scala.concurrent.ExecutionContext
-
-/*
-  WARNING: This test suite is designed to only be run locally in order to trigger real email notifications to the
-  anghammarad.test.alerts Google group. This can be useful for viewing changes to the email format, for example.
-
-  In order to utilise this you can remove the 'DoNotDiscover` annotation on the class and you may
-  choose to replace 'ignore' with 'in' in the test cases below, as appropriate, and then run the tests as usual.
- */
-@DoNotDiscover
-class SendTestNotifications extends FreeSpec with OneAppPerSuiteWithComponents with Matchers with AttemptValues {
```

Comment:
Yeah, I can see that this would be useful! I'm not crazy about the approach though, so let's talk about what is useful here. Generally for this kind of irregular manually triggered task we're better having a CLI endpoint to trigger it somewhere, rather than putting it in our tests.